app/templates/services/previous_services.html

```diff
@@ -0,0 +1,89 @@
+{% extends "_base_page.html" %}
+{% import "toolkit/summary-table.html" as summary %}
+{% import "macros/submission.html" as submission %}
+
+{% block page_title %}Copy your {{ framework.name }} services – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace",
+      },
+      {
+        "link": url_for(".dashboard"),
+        "label": "Your account",
+      },
+      {
+        "link": url_for(".framework_dashboard", framework_slug=framework.slug),
+        "label": "Apply to " + framework.name,
+      },
+      {
+        "link": url_for(".framework_submission_lots", framework_slug=framework.slug),
+        "label": "Services",
+      },
+      {
+        "link": url_for(".framework_submission_services", framework_slug=framework.slug, lot_slug=lot.slug),
+        "label": lot.name,
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+  {% include "partials/service_warning.html" %}
+
+  {% if copy_all %}
+    <form action="{{ url_for('.copy_all_previous_services', framework_slug=framework.slug, lot_slug=lot.slug) }}" method="POST">
+      <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
+      {%
+        with
+        message = "Are you sure you want to add all your {} services?".format(lot.name|lower),
```

Comment:
Can we show this above the `partials/service_warning` so that both green boxes (the request for confirmation and the flash message) appear above the blue box. Otherwise it's a bit weird - the confirmation is below and the flash is above.